index.php

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Movie</title>
+</head>
+<body>
+    <div class="homebar">
+        <h1 class="homepage">Welcome To My Movie Database</h1>
+</div>
+<div class="navbar">
+        <a class="home" href="http://localhost:1234/Collection-App/">Home</a>
+        <a class="Genre" href="#">Genre</a>
+        <a class="Add-title" href="#">Add Movie</a>
```

Comment:
For now, you should take these two out - we never want to have code that doesn't work, including links that don't go anywhere

Reply:
I have taken away the genre tab and kept the add title to use as the add a title to my database